sdk/cosmos/azure-cosmos-encryption/pom.xml

```diff
@@ -37,6 +37,10 @@ Licensed under the MIT License.
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <jacoco.min.linecoverage>0.01</jacoco.min.linecoverage>
     <jacoco.min.branchcoverage>0.01</jacoco.min.branchcoverage>
+
+    <!-- CosmosSkip - This is not a module we want/expect external customers to consume. Skip breaking API checks. -->
+    <!-- This can only be enabled once we release GA, as it needs a stable version to check for breaking changes. -->
+    <revapi.skip>true</revapi.skip>
```

Comment:
azure-cosmos-encryption is going to be customer-facing and it shouldn't skip revapi checks. This library is still in preview and skipping revapi is not required anyway I guess.

cc: @kushagraThapar